dataflow/gpu-workers/pytorch-minimal/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM pytorch/pytorch:1.8.1-cuda11.1-cudnn8-runtime
+
+WORKDIR /pipeline
+
+# Copy the Apache Beam worker files and the pipeline source files.
+COPY --from=apache/beam_python3.8_sdk:2.29.0 /opt/apache/beam /opt/apache/beam
+COPY requirements.txt .
+COPY *.py ./
+
+# Install the pipeline requirements and check that there are no conflicts.
+RUN pip install --no-cache-dir --upgrade pip \
+    && pip install --no-cache-dir -r requirements.txt \
```

Comment:
We could make a comment somewhere that since users already install requirements.txt in the dockerfile, they should not pass --requirements_file when they launch the pipeline.